lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -174,7 +175,7 @@ Future<void> initializeEngineServices({
     // fires.
     if (!waitingForAnimation) {
       waitingForAnimation = true;
-      domWindow.requestAnimationFrame(allowInterop((num highResTime) {
+      domWindow.requestAnimationFrame((JSNumber highResTime) {
```

Comment:
In other JS-interop layers, I've tried to prevent the JSNumber from leaking to users by wrapping the user-supplied callback in something that gets called from the js-interop layer after making the transformation?

Something like this maybe?

```
  double requestAnimationFrame(DomRequestAnimationFrameCallback callback) =>
      _requestAnimationFrame((JSNum n) {
        callback(n.toDart);
      }.toJS).toDart;
```

Reply:
oh we can do that. It does mean an extra closure. Do you prefer that approach?

Reply:
This particular case is called very often, so the faster, the better (so I guess: the fewer indirections, the better?)